### DISCUSSION TITLE
Help with website or docs?

### DISCUSSION THREAD

**Discussion by rayosdev** [friction2d/friction#337] [Q&A]
I'm a web / game developer and wanted to know how to best contribute with web or documentation if that is wanted

**Comment by pgilfernandez** [comment]
I would say the most needed work it to have a basic documentation.
I would start porting old Enve ones:
- https://theflydesign.es/manual-enve-2d-animation-english/
- https://maurycyliebner.github.io/docs/shaderEffects

But I don't know if @rodlie wants to first of all propose an index so that we can start filling them up

**Reply by rodlie** [reply]
The website is not that important at the moment, I will do some changes before v1.0 final. We can discuss the website after that.

Documentation is more important, I will make this a priority after the v1.0 beta2 release, as I want basic documentation ready for v1.0 RC1.

**Reply by pgilfernandez** [reply]
Yes, that makes sense.
I hope to help with some videos and docs...

**Comment by pgilfernandez** [comment]
@rodlie, could you give us some little information about how could be start creating docs, sections, blablabla?
I have pretty much experience with html, css, js, etc. but never used Jekyll...

Thanks

**Reply by rodlie** [reply]
I will add some info later, busy with beta 2 testing/release notes.

It all just markdown, you should not need to think about the backend ;)

**Reply by rodlie** [reply]
btw, release notes for beta2: https://github.com/friction2d/friction2d.github.io/blob/main/_drafts/friction-100-beta2.md

**Reply by pgilfernandez** [reply]
Yes, I figured that out from the researching the repository but I wanted to know just case and to know how to test it locally.

I Will Start doing some mockups locally in mansions and will upload when I learn how to do it.

Focus on b2 😍

**Reply by rodlie** [reply]
The text is the important part, we can figure out the rest later.

**Reply by rodlie** [reply]
btw @pgilfernandez I added one of your follow path SVG's and your sub-path offset mp4 in the release notes, I hope you don't mind.

* https://friction.graphics/assets/screenshots/100/friction-follow-path.svg
* https://friction.graphics/assets/screenshots/100/friction-subpath-offset.mp4

![localhost_4000_releases_friction-100-beta2 html (2)](https://github.com/user-attachments/assets/ce225aaa-6783-43c4-bcec-1b1b72068406)

**Reply by pgilfernandez** [reply]
> btw @pgilfernandez I added one of your follow path SVG's and your sub-path offset mp4 in the release notes, I hope you don't mind.

Noooo, at the contrary, I'm happy it helped you document it faster.
Next time I could design something cuter, hehehe

**Reply by rodlie** [reply]
> Next time I could design something cuter

We always have the final release notes :wink: I usually don't put much effort into beta release notes :smile: Good Enough™
